docs/best_practices.md

```diff
@@ -0,0 +1,94 @@
+Since 2014 when Force was first open sourced, Artsy.net's underlying [technology choices have evolved](https://artsy.github.io/blog/2018/10/04/artsy-frontend-history/). As our understanding grows, newer code uses newer techniques. Older code is often left un-updated. It can be difficult to orient oneself around what the current preferred practices are.
+
+This document is a map. Not of Force at a specific time, but a map of how we got here and where we want to go next. This is a living document, expected to be updated regularly, of links to:
+
+- Example code.
+- Pull requests with interesting discussions.
+- Conversations on Slack.
+- Blog posts.
+
+Links should point to specific commits, and not a branch (in case the branch or file is deleted, these links should always work). But it's possible that a file is outdated, that our understanding has moved on since it was linked to; in that case, please update this document.
+
+## Current Preferred Practices
+
+The app is currently written responsively in React, with an ever-increasing amount of legacy code written in Backbone and Coffeescript. As of Nov 2020, most of our main pages have been migrated, and those [that remain become fewer](https://github.com/artsy/force/pull/6682) with time. Server-side code is built on top of Express.js; however, most server-side needs have been abstracted away by our framework (see below).
+
+### When writing UI, use Palette
+
+Our design system [Palette](https://github.com/artsy/palette) is used for most UI needs. Check out [the docs](https://palette.artsy.net/)!
+
+> If interested in some of the lower-level particulars of how Palette was built, see the [styled-system](https://styled-system.com/) docs.
+
+### For routing, use our framework
+
+Individual sub-apps (represented by routes like `/artist/:id` or `/collect`) are built on top of a home-grown SSR (server-side-rendering) framework based around [Found](https://github.com/4Catalyzer/found), a routing library. Sub-apps are mounted within the [global routes file](https://github.com/artsy/force/blob/1842553ad34475bc3b804f00c6410d7f23d64f65/src/v2/Apps/getAppRoutes.tsx).
+
+To learn how to create a new sub-app, see [the docs](https://github.com/artsy/force/blob/1842553ad34475bc3b804f00c6410d7f23d64f65/docs/adding_new_app.md).
```

Comment:
Is this intentionally a permalink to an old version of this document, or should it link to the evergreen version?

```suggestion
To learn how to create a new sub-app, see [the docs](https://github.com/artsy/force/blob/master/docs/adding_new_app.md).
```

Reply:
It was intentional to link to something concrete. Having a link that might be slightly outdated (but representative within the context of the surrounding documentation) is preferrable to a 404 (which happens frequently). I think as this evolves these links will evolve too (when necessary).